cli/src/main.rs

```diff
@@ -1163,6 +1163,77 @@ fn run<I: IntoIterator<Item = T>, T: Into<OsString> + Clone>(args: I) -> Result<
                                 .help("A role to be assigned to the provided identity"),
                         ),
                 )
+                .subcommand(
+                    SubCommand::with_name("update")
+                        .about("Updates an authorized identity on a Splinter node")
+                        .arg(
+                            Arg::with_name("url")
+                                .short("U")
+                                .long("url")
+                                .help("URL of the Splinter daemon REST API")
+                                .takes_value(true),
+                        )
+                        .arg(
+                            Arg::with_name("private_key_file")
+                                .value_name("private-key-file")
+                                .short("k")
+                                .long("key")
+                                .takes_value(true)
+                                .help("Name or path of private key"),
+                        )
+                        .arg(
+                            Arg::with_name("id_key")
+                                .value_name("public-key")
+                                .long("id-key")
+                                .takes_value(true)
+                                .required(true)
+                                .conflicts_with("id_user")
+                                .help("The public key identity being assigned roles"),
+                        )
+                        .arg(
+                            Arg::with_name("id_user")
+                                .value_name("user-id")
+                                .long("id-user")
+                                .takes_value(true)
+                                .required(true)
+                                .conflicts_with("id_key")
+                                .help("The user identity being assigned roles"),
+                        )
+                        .arg(
+                            Arg::with_name("add_role")
+                                .value_name("role")
+                                .long("add-role")
+                                .takes_value(true)
+                                .multiple(true)
+                                .number_of_values(1)
+                                .help("A role to be added to the provided identity's assignments"),
+                        )
+                        .arg(
+                            Arg::with_name("force")
+                                .short("f")
+                                .long("force")
+                                .help("Ignore errors, such as adding and removing the same value."),
+                        )
+                        .arg(
+                            Arg::with_name("rm_role")
+                                .value_name("role")
```

Comment:
Is this `value_name("role")` intentional? Looks like it's the same as the one for `add-role`.

Reply:
This is the value displayed in the help: `--rm-role <role>`